sentry-ruby/lib/sentry/rake.rb

```diff
@@ -19,9 +19,7 @@ module Task
       def execute(args=nil)
         return super unless Sentry.initialized? && Sentry.get_current_hub
 
-        Sentry.get_current_hub.with_background_worker_disabled do
```

Comment:
Looking at `execute` here — isn't it just not doing anything in this form? Looks like it falls back to `super` in every branch, no? /cc @st0012